runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/Pet.java

```diff
@@ -84,15 +85,18 @@
 	private final String name;
 	private final Integer iconID;
 
-	static Pet findPet(String petName)
+	private static final HashMap<String, Pet> PET_HASH_MAP = new HashMap<>();
```

Comment:
There's only ~40 something pets, this really isn't necessary. And also we typically use `ImmutableMap` for stuff like this.